angular/official/blank/src/app/pages/home/home.page.scss

```diff
@@ -1 +1,3 @@
-// homepage styles
+app-page-home {
```

Comment:
This selector is not required. Angular CSS pulled in from `styles` or `styleUrls` are already scoped: https://angular.io/guide/component-styles#style-scope